runs/senddebug.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
-echo "***** debuglog start..." >> /var/www/html/openWB/ramdisk/openWB.log
+echo "***** debuglog level 1 start..." >> /var/www/html/openWB/ramdisk/openWB.log
 sed -i 's/debug.*/debug=1/' /var/www/html/openWB/openwb.conf
 sleep 60
+echo "***** debuglog level 2 start..." >> /var/www/html/openWB/ramdisk/openWB.log
+sed -i 's/debug.*/debug=2/' /var/www/html/openWB/openwb.conf
```

Comment:
Wenn jemand ein Smarthome-Device aus irgend einem Grund "debug" nennen sollte wird das Gerät hiermit umbenannt. Wenn es noch eine Option `whateverdebugfoo` in der Konfig von irgendwas anderem gibt, wird die Option geändert. Besser wäre `s/^debug=.*/debug=2/`. (Betrifft natürlich auch Zeile 3)

Reply:
Guter Hinweis, hatte ich übersehen. Habe mal schnell nach weiteren betroffenen Skripts gesucht und angepasst.